### PR TITLE
i18n: use WordPress `%s` placeholders

### PR DESCRIPTION
**Pull request by ramiy**
The current translation strings use `%subject`, `%email1, and `%email2` placeholders.

Those custom placeholders are not recognized by Transifex, making it harder to translate the strings.

This PR updates the placeholders to the standard `%s`, `%1$s`, `%2$s` placeholders used by WordPress core.

### CODE REVIEW

File: assets/js/src/common/sender_email_address_warning.jsx [hunk 1]
```diff
@@ -21,10 +21,10 @@ const SenderEmailAddressWarning = ({ emailAddress, mssActive }) => {
         <p className="sender_email_address_warning">
           {ReactStringReplace(
             MailPoet.I18n.t('senderEmailAddressWarning2'),
-            /(%suggested|%originalSender|<em>.*<\/em>)/,
+            /(%1$s|%2$s|<em>.*<\/em>)/,
```
Comment (NeosinneR) [inline assets/js/src/common/sender_email_address_warning.jsx]: Hey ramiy, thanks for PR. One small issue, these two need to be escaped - `%1$s|%2$s` => `%1\$s|%2\$s`

Comment (ramiy) [reply assets/js/src/common/sender_email_address_warning.jsx]: Done.

**Comment by codemonkey-jack** [comment]
@NeosinneR Can you check this, please? It has conflicts that need solving before it can be merged. Thanks!

**Comment by ramiy** [comment]
@codemonkey-jack The conflict is because of #2277

**Comment by ramiy** [comment]
@codemonkey-jack Conflicts resolved. Please review again.

**Comment by NeosinneR** [comment]
@ramiy could you please rebase this branch on `master` instead of merging it? Github seems to be a bit confused and still sees a conflict

**Comment by codemonkey-jack** [comment]
@ramiy When you get a chance could you take a look at the conflicts, rebasing this branch on `master`, please? Thanks!

**Comment by codemonkey-jack** [comment]
Hey @ramiy, can you rebase from master, please? We can then merge it. 

To avoid stale pull requests we'll close this on Monday if it isn't mergeable by that point. 

Thanks!

**Pull request closed without merging**